app/src/main/res/xml/prefs.xml

```diff
@@ -1,75 +1,90 @@
 <?xml version="1.0" encoding="utf-8"?>
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.preference.PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:key="@string/preference_key_screen">
 
-    <PreferenceCategory
+    <androidx.preference.PreferenceCategory
         android:key="@string/preference_key_category_general"
-        android:title="@string/general_settings">
+        android:title="@string/general_settings"
+        app:iconSpaceReserved="false">
 
-        <CheckBoxPreference
+        <androidx.preference.SwitchPreferenceCompat
             android:defaultValue="@bool/preference_default_value_auto_update_enabled"
             android:key="@string/preference_key_auto_update_enabled"
-            android:title="@string/preference_title_auto_update_enabled" />
+            android:title="@string/preference_title_auto_update_enabled"
+            app:iconSpaceReserved="false" />
 
-        <Preference
+        <androidx.preference.Preference
             android:key="@string/preference_key_app_notification_settings"
-            android:title="@string/preference_title_app_notification_settings" />
+            android:title="@string/preference_title_app_notification_settings"
+            app:iconSpaceReserved="false" />
 
-        <EditTextPreference
+        <!--
+        TODO: Use ValidateableEditTextPreference
+        see https://github.com/EventFahrplan/EventFahrplan/issues/62
+        Note: android:hint is ignored. See https://issuetracker.google.com/issues/37060038
+        -->
+        <androidx.preference.EditTextPreference
```

Comment:
This could be `StyleableEditTextPreference` until issue #62 is implemented.

Reply:
:heavy_check_mark: Changed in 23b10ca4822a133ed18915842987e132b4a9205a but styles stopped working now.